frontend/pages/index.vue

```diff
@@ -1,23 +1,50 @@
 <template>
   <v-container>
     <v-row>
-      <v-col cols="12" class="dashboard-grid">
-        <EquipmentCard
-          v-for="equipment in equipments"
-          :key="equipment.id"
-          :equipment="equipment"
-        />
+      <v-col v-if="activeEquipment" cols="10" class="border-e-md">
```

Comment:
I think we discussed having a separate **page** for the equipment details. I suggest reverting this dashboard page back to where it was, and making a separate page with the equipment details. I reiterate this since there shouldn't be any value to the customer to look at other equipment while they're looking at the details for another.